sdk/python/setup.py

```diff
@@ -196,7 +196,7 @@ def finalize_options(self):
 
     def _generate_python_protos(self, path: str):
         proto_files = glob.glob(os.path.join(self.proto_folder, path))
-
+        Path(self.python_folder).mkdir(exist_ok=True)
```

Comment:
any particular reason for adding this? `self.python_folder` is set to `feast/sdk/python/feast/protos`, which already exists as a directory in our git repo

Reply:
I had deleted the directory locally - i think you're right that this isn't needed.